.github/workflows/ci.yml

```diff
@@ -110,6 +114,10 @@ jobs:
       matrix:
         arch: [x86, x64]
     steps:
```

Comment:
not your fault, but this "windows-vc14" run isn't doing what it claims.

vc14 means version 14.0 of the compiler, aka toolset 140, which came with Visual Studio 2015, but you can still use those older toolset versions with later versions of Visual Studio

but it's actually just using the default compiler on this machine, which is Visual Studio 2019's version 14.29

Here's the [line in the logs](https://github.com/awslabs/aws-crt-dotnet/actions/runs/4008641738/jobs/6883066390#step:4:174) where you see what it's actually using.
>   -- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x86/cl.exe

and then you cross-reference that against the [Wikipedia page that details what the hell those version numbers mean](https://en.wikipedia.org/wiki/Microsoft_Visual_C%2B%2B#Internal_version_numbering). And you see that 14.29 means it's the latest build Visual Studio 2019 compiler

Reply:
Added a TODO for now.